src/components/timeline/Extensionboth.jsx

```diff
@@ -0,0 +1,402 @@
91 0.6 60.6 0.779086 60.6 1C60.6 1.22091 60.7791 1.4 61 1.4V0.6ZM69 1.4C69.2209 1.4 69.4 1.22091 69.4 1C69.4 0.779086 69.2209 0.6 69 0.6V1.4ZM77 0.6C76.7791 0.6 76.6 0.779086 76.6 1C76.6 1.22091 76.7791 1.4 77 1.4V0.6ZM85 1.4C85.2209 1.4 85.4 1.22091 85.4 1C85.4 0.779086 85.2209 0.6 85 0.6V1.4ZM93 0.6C92.7791 0.6 92.6 0.779086 92.6 1C92.6 1.22091 92.7791 1.4 93 1.4V0.6ZM101 1.4C101.221 1.4 101.4 1.22091 101.4 1C101.4 0.779086 101.221 0.6 101 0.6V1.4ZM109 0.6C108.779 0.6 108.6 0.779086 108.6 1C108.6 1.22091 108.779 1.4 109 1.4V0.6ZM117 1.4C117.221 1.4 117.4 1.22091 117.4 1C117.4 0.779086 117.221 0.6 117 0.6V1.4ZM125 0.6C124.779 0.6 124.6 0.779086 124.6 1C124.6 1.22091 124.779 1.4 125 1.4V0.6ZM133 1.4C133.221 1.4 133.4 1.22091 133.4 1C133.4 0.779086 133.221 0.6 133 0.6V1.4ZM141 0.6C140.779 0.6 140.6 0.779086 140.6 1C140.6 1.22091 140.779 1.4 141 1.4V0.6ZM149 1.4C149.221 1.4 149.4 1.22091 149.4 1C149.4 0.779086 149.221 0.6 149 0.6V1.4ZM157 0.6C156.779 0.6 156.6 0.779086 156.6 1C156.6 1.22091 156.779 1.4 157 1.4V0.6ZM165 1.4C165.221 1.4 165.4 1.22091 165.4 1C165.4 0.779086 165.221 0.6 165 0.6V1.4ZM173 0.6C172.779 0.6 172.6 0.779086 172.6 1C172.6 1.22091 172.779 1.4 173 1.4V0.6ZM181 1.4C181.221 1.4 181.4 1.22091 181.4 1C181.4 0.779086 181.221 0.6 181 0.6V1.4ZM189 0.6C188.779 0.6 188.6 0.779086 188.6 1C188.6 1.22091 188.779 1.4 189 1.4V0.6ZM197 1.4C197.221 1.4 197.4 1.22091 197.4 1C197.4 0.779086 197.221 0.6 197 0.6V1.4ZM205 0.6C204.779 0.6 204.6 0.779086 204.6 1C204.6 1.22091 204.779 1.4 205 1.4V0.6ZM213 1.4C213.221 1.4 213.4 1.22091 213.4 1C213.4 0.779086 213.221 0.6 213 0.6V1.4ZM221 0.6C220.779 0.6 220.6 0.779086 220.6 1C220.6 1.22091 220.779 1.4 221 1.4V0.6ZM229 1.4C229.221 1.4 229.4 1.22091 229.4 1C229.4 0.779086 229.221 0.6 229 0.6V1.4ZM237 0.6C236.779 0.6 236.6 0.779086 236.6 1C236.6 1.22091 236.779 1.4 237 1.4V0.6ZM245 1.4C245.221 1.4 245.4 1.22091 245.4 1C245.4 0.779086 245.221 0.6 245 0.6V1.4ZM253 0.6C252.779 0.6 252.6 0.779086 252.6 1C252.6 1.22091 252.779 1.4 253 1.4V0.6ZM261 1.4C261.221 1.4 261.4 1.22091 261.4 1C261.4 0.779086 261.221 0.6 261 0.6V1.4ZM269 0.6C268.779 0.6 268.6 0.779086 268.6 1C268.6 1.22091 268.779 1.4 269 1.4V0.6ZM277 1.4C277.221 1.4 277.4 1.22091 277.4 1C277.4 0.779086 277.221 0.6 277 0.6V1.4ZM285 0.6C284.779 0.6 284.6 0.779086 284.6 1C284.6 1.22091 284.779 1.4 285 1.4V0.6ZM293 1.4C293.221 1.4 293.4 1.22091 293.4 1C293.4 0.779086 293.221 0.6 293 0.6V1.4ZM301 0.6C300.779 0.6 300.6 0.779086 300.6 1C300.6 1.22091 300.779 1.4 301 1.4V0.6ZM1 1.4H5V0.6H1V1.4ZM13 1.4H21V0.6H13V1.4ZM29 1.4H37V0.6H29V1.4ZM45 1.4H53V0.6H45V1.4ZM61 1.4H69V0.6H61V1.4ZM77 1.4H85V0.6H77V1.4ZM93 1.4H101V0.6H93V1.4ZM109 1.4H117V0.6H109V1.4ZM125 1.4H133V0.6H125V1.4ZM141 1.4H149V0.6H141V1.4ZM157 1.4H165V0.6H157V1.4ZM173 1.4H181V0.6H173V1.4ZM189 1.4H197V0.6H189V1.4ZM205 1.4H213V0.6H205V1.4ZM221 1.4H229V0.6H221V1.4ZM237 1.4H245V0.6H237V1.4ZM253 1.4H261V0.6H253V1.4ZM269 1.4H277V0.6H269V1.4ZM285 1.4H293V0.6H285V1.4ZM301 1.4H305V0.6H301V1.4Z" fill="#C4C4C4" />
```

Comment:
Also keep this in content

Reply:
It was hard to handle such so just going on with img that doesn't affect anything